projects/2022_03_03_FasterCrypt_Competition.md

```diff
@@ -0,0 +1,80 @@
+# FasterCrypt Competition
+FasterCrypt is a competition to optimize 3 encryption algorithms which are used in the PacketCrypt bandwidth hard proof of work. These encryption algorithms are also used in many other software programs so the results of this competition will benefit the entire internet ecosystem.
+
+* Contact Email: tomas@leitecastro.com
+* Project participants:
+  * Tomás Leite de Castro (tomas@leitecastro.com and @tomas347 - https://pkt.chat)
+  * Judges 
+  * Contestants
+* I ask the network stewards a contribution of the value equivalent of 17 000€ to help me cover the costs associated with hosting the competition. I will also have to develop the test harness, and use the hardware to benchmark the competitors code. (This value was estimated considering the competition will be hosted for 3 months, I intend to return the rest if the competition ends sooner). I only ask for my PKT contribution to be paid by the NS after the competition is over, in order to reduce risks. 
```

Comment:
These are my personal opinions and not the words of the Network Steward:

You really need to build on the template for this. You're missing specific milestones with defined success criteria for their completion and that's going to make it hard for the NS, if it accepts this project, to validate whether it was delivered properly. The templates have everything included that you need. Just remember that it is a much stronger project if you:
1. Spec every deliverable in a stable currency, while
2. Establishing a maximum budget defined in PKT